static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -429,3 +428,11 @@ const RowHeader = styled('span')<{expandable: boolean; expanded: boolean}>`
     min-height: 32px;
   }
 `;
+
+const StackTraceFrame = styled('li')`
+  :not(:last-child) {
+    ${RowHeader} {
+      border-bottom: 1px solid ${p => p.theme.border};
```

Comment:
this fixes the double border added in the PR https://github.com/getsentry/sentry/pull/44284

**Current:**

![image](https://user-images.githubusercontent.com/29228205/229789522-b782d0fd-114c-4da4-b73e-ddd1d5a93a05.png)


**With this style changes:**

![image](https://user-images.githubusercontent.com/29228205/229789734-7a40cf9d-a58a-4b78-9741-dc021a59a515.png)